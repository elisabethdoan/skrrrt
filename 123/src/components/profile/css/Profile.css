img {
    width: 30rem;
    border: 0.4rem solid #E2432A;
    border-radius: 4px;
    padding: 5px;
}

#profile {
    display: flex;
    flex-direction: column;
    align-items: center;
    margin-bottom: 6vh;
}

@media only screen and (max-width: 600px) {
    img {
        width: 15rem;
        border: 0.2rem solid #E2432A;
        border-radius: 2px;
        padding: 3px;
    }
}