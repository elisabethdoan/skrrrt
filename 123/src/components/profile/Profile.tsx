import React from 'react';
import './css/Profile.css';

const Profile: React.FC = () => (
  <div id="profile">
    <h4>IT2810-Prosjekt2</h4>
    <img
      src="/gitlab_avatar.jpeg"
      alt="gitlab_avatar"
    />
  </div>
);

export default Profile;
