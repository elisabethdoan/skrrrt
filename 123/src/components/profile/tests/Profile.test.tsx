import { cleanup, render } from "@testing-library/react";
import React from "react";
import Profile from '../Profile';
import renderer from 'react-test-renderer';

describe('Navigation button', () => {
    
    afterEach(cleanup)
  
    test('Renders with correctly', () => {
        const tree = renderer.create(<Profile />).toJSON();
        expect(tree).toMatchSnapshot();

    })
  
  });