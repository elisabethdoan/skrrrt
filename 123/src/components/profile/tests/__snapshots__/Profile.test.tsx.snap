// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation button Renders with correctly 1`] = `
<div
  id="profile"
>
  <h4>
    IT2810-Prosjekt2
  </h4>
  <img
    alt="gitlab_avatar"
    src="/gitlab_avatar.jpeg"
  />
</div>
`;
